zkevm-circuits/src/circuit_tools/cell_manager.rs

```diff
@@ -274,16 +288,87 @@ impl<F: Field, C: CellType> CellManager<F, C> {
                     expr: cells[0].expr(),
                     cells,
                 });
+                width += 1;
```

Comment:
`width` is just `columns.len()`? Why the extra variable?

Reply:
probably because in previous version there was a `width`? I'll get rid of it :)